examples/pipelines/simple_pipeline/components/image_filtering/Dockerfile

```diff
@@ -5,23 +5,13 @@ RUN apt-get update && \
     apt-get upgrade -y && \
     apt-get install git curl -y
 
-# Downloading gcloud package
-RUN curl https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz > /tmp/google-cloud-sdk.tar.gz
-
-# Installing the package
-RUN mkdir -p /usr/local/gcloud \
-&& tar -C /usr/local/gcloud -xvf /tmp/google-cloud-sdk.tar.gz \
-&& /usr/local/gcloud/google-cloud-sdk/install.sh
-
-# Adding the package path to local
-ENV PATH $PATH:/usr/local/gcloud/google-cloud-sdk/bin
-
 # install requirements
 COPY requirements.txt /
 RUN pip3 install --no-cache-dir -r requirements.txt
 
 # Copy over src-files of the component
 COPY src /src
+COPY fondant_component.yaml /src
```

Comment:
We should keep the same directory structure in the docker image so we can run the same code both locally and in the docker image.